apps/prairielearn/src/pages/authLogout/authLogout.ts

```diff
@@ -0,0 +1,37 @@
+import { Router } from 'express';
+import asyncHandler = require('express-async-handler');
+import util = require('util');
+import { config } from '../../lib/config';
+
+const router = Router();
+
+router.get(
+  '/',
+  asyncHandler(async (req, res, _next) => {
```

Comment:
Consider opening another PR with the async refactor + `req.session.destroy` call so that Git will maintain the identity of this file before + after the extension change.

Reply:
Yeah, I can make a new PR that does better to persist the file. My `git rm` and `git add` should probably have been a `git mv`. I should be able to do the rename and edits in the same PR, but I didn't set this one up this way.

Reply:
OK, I will do the functionality changes here and the TS conversion in a different PR.